src/constants/constants.tsx

```diff
@@ -20,3 +20,8 @@ export enum ERROR_HANDLING_STRATEGY {
   Stop = 'stop',
   DeadLetterQueue = 'dead_letter_queue',
 }
+
+export const ADDON_INSTALLATION_GUIDE_URL =
```

Comment:
I wonder, I've stored URLs like this in the i18n files in the past.  It was an approach that was kind of convenient as it allowed other folks to update the URLs as these tend to change over time external to the app.  Not sure how much that approach really helps us here though.  But something to be mindful of I guess, maybe at some point we figure out a nice way to externalize these and expose them in an easy to manage way.

Reply:
_kind of convenient as it allowed other folks to update the URLs as these tend to change over time external to the app._  

@gashcrumb I didn't follow. since the translation file is also part of the UI build and loaded from the same CDN package. am I missing something?